SE_Lab10/Source.cpp

```diff
@@ -0,0 +1,95 @@
+#include <algorithm>
+#include <iostream>
+#include <vector>
+#include <string.h>
+
+
+class  Concate {
+public:
+	char first[15] = "first part,";
+	char second[15] = "second part";
+
+	friend void myfun(Concate obj);
+};
+
+void myfun(Concate obj) {
+	strcat_s(obj.first, obj.second);
+	std::cout << obj.first;
+}
+
+
+auto lambda = [](int x) {
+	return (x > 0) ? true : false;
+	};
+
+
+auto check = [](auto val) {
+	return std::is_same_v<decltype(val), char >&& val > 0 && val < 128;
+	};
+
+
+auto conct = [](const std::string& one, const std::string& two) {
+	return one + two;
+	};
+
+
+auto longerStr = [](const std::string& str1, const std::string& str2) {
+	return(str1.length() > str2.length()) ? str1 : str2;
+	};
+
+
+int main() {
```

Comment:
change one style for your code, for example part of code has few new line but another part none